SU-HSDO/suhumsci
# shs-109: refactor page scroll animation

## Summary
Refactors the page scroll animation JS

## Steps to Test
1. On both Colorful and Traditional, go to `/components/hero-images-main-region` and the homepage and ensure that the heroes that used to animate still animate correctly.

**Notes**
I kept the preexisting array of items watch for animation. When reducing it to the parent component, I discovered that different animations fire based off image, text, etc. so most of the classes all relate to two parent elements (the overlay and gradient heroes). I also kept the ones I wasn't sure (`hs-font-lead`, `hs-font-splash`) because I can see they animate things in the code, but I don't know where they exist.
